ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/ExceptionManagement.helpers.ts

```diff
@@ -14,11 +14,24 @@ export const approvedDeferralsPath = `${basePath}/approved-deferrals`;
 export const approvedFalsePositivesPath = `${basePath}/approved-false-positives`;
 export const deniedRequestsPath = `${basePath}/denied-requests`;
 
-export function visitExceptionManagement() {
+export function visitPendingRequestsTab() {
     visit(pendingRequestsPath);
 
     cy.get('h1:contains("Exception management")');
     cy.location('pathname').should('eq', pendingRequestsPath);
+
+    // Wait for the loading spinner to disappear
+    cy.get('.pf-c-spinner').should('not.exist');
```

Comment:
Two possible ideas for consideration:

- In addition to asserting that the spinner does not exist, assert that it _does_ exist first. (Would it be possible for this assertion to run after the `h1` is rendered but before the table spinner is rendered?)
- Scope the spinner element more closely, e.g. to a spinner beneath a `table`, so that multiple spinners on a page would not cause this check to start failing